File: HELLO_PYTHON/day06/mysql_select.py
import pymysql
 
conn = pymysql.connect(host='localhost', user='root', password='python',
                       port=3305, db='python', charset='utf8')
 
curs = conn.cursor(pymysql.cursors.DictCursor)
 
sql = "select * from emp"
curs.execute(sql)
 
rows = curs.fetchall()
print(rows)

curs.close()
conn.close()

File: HELLO_NODE/mysql_select.js
const mysql = require('mysql');
const connection = mysql.createConnection({
	host: 'localhost',
	port: 3305,
	user: 'root',
	password: 'python',
	database: 'python'
});

connection.connect();

connection.query('SELECT * from emp', (error, rows, fields) => {
	console.log(rows);
});

connection.end();